app/api/v1/endpoints/email_accounts.py

```
@@ -0,0 +1,76 @@
+    name = get_settings().tenant_header_name
+    if not x_tenant_id:
+        raise HTTPException(status_code=400, detail=f"Missing required header: {name}")
+    return x_tenant_id
+
+
+@router.get("")
+async def list_email_accounts(
+    tenant_id: Annotated[str, Depends(_tenant_id)],
+    skip: int = 0,
+    limit: int = 100,
+    email_account_repo: EmailAccountRepository = Depends(get_email_account_repo),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)